app_flutter/test/utils/golden.dart

```diff
@@ -18,5 +23,24 @@ Future<void> expectGoldenMatches(
   dynamic skip = false, // true or a String
 }) {
   final String goldenPath = path.join('goldens', goldenFileKey);
+  goldenFileComparator = CocoonFileComparator(Uri.parse(goldenPath));
   return expectLater(actual, matchesGoldenFile(goldenPath), reason: reason, skip: skip || !Platform.isLinux);
 }
+
+class CocoonFileComparator extends LocalFileComparator {
+  CocoonFileComparator(Uri testFile) : super(testFile);
+
+  @override
+  Future<bool> compare(Uint8List imageBytes, Uri golden) async {
+    final ComparisonResult result = await GoldenFileComparator.compareLists(
+      imageBytes,
+      await getGoldenBytes(golden),
+    );
+
```

Comment:
Can you log a warning when diffPercent > 0? This helps up notice gold diff creep before it breaks the tree